wasm/jit/jit_amd64.go

```diff
@@ -1096,6 +1096,30 @@ func (b *amd64Builder) handleLe(o *wazeroir.OperationLe) error {
 	return nil
 }
 
+func (b *amd64Builder) handleMemoryGrow() {
+	b.callBuiltinFunctionFromConstIndex(builtinFunctionIndexMemoryGrow)
+}
+
+func (b *amd64Builder) handleMemorySize() {
+	b.callBuiltinFunctionFromConstIndex(builtinFunctionIndexMemorySize)
+	loc := b.locationStack.pushValueOnStack() // The size is pushed on the top.
+	loc.setValueType(wazeroir.SignLessTypeI32)
+}
+
+func (b *amd64Builder) callBuiltinFunctionFromConstIndex(index int64) {
+	// Set the jit status as jitCallStatusCodeCallBuiltInFunction
+	b.setJITStatus(jitCallStatusCodeCallBuiltInFunction)
+	// Set the function index.
+	b.setFunctionCallIndexFromConst(index)
+	// Release all the registers as our calling convention requires the callee-save.
+	b.releaseAllRegistersToStack()
+	// Set the continuation offset on the next instruction.
+	b.setContinuationOffsetAtNextInstructionAndReturn()
+	// Once we return from the function call,
+	// we must setup the reserved registers again.
+	b.initializeReservedRegisters()
```

Comment:
I like the intent to comment, but the only meaningful ones that say something more than what the function names say are.. 

```
// Release all the registers as our calling convention requires the callee-save.
```
and

```
Once we return from the function call,
 	// we must setup the reserved registers again.
```

can you remove the redundant ones as all lines are debt even comments unless they perform some good :D

Reply:
👍  https://github.com/tetratelabs/wazero/pull/85/commits/1b6114f43181c5f4c39e34e54a9c0cb86c90ab4f